databrowser/src/domain/datasets/ui/toolBox/export/ExportDatasetsToolBoxPanel.vue

```diff
@@ -5,102 +5,112 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 -->
 
 <template>
-  <ToolBoxPanel>
-    <ToolBoxSectionLabel>{{
-      t('datasets.toolBox.exportDatasets.sectionRetrieveData')
-    }}</ToolBoxSectionLabel>
-    <ToolBoxCard v-if="url != null">
-      <ToolBoxCardHeader>
-        {{ t('datasets.toolBox.exportDatasets.apiDatasets.header') }}
-        <ToolBoxCardHeaderButton
+  <ToolBoxSectionLabel>
+    {{ t('datasets.toolBox.exportDatasets.sectionRetrieveData') }}
+  </ToolBoxSectionLabel>
+
+  <ToolBoxCard v-if="url != null">
+    <ToolBoxCardHeader :uppercase="false" class="rounded">
+      {{ t('datasets.toolBox.exportDatasets.apiDatasets.header') }}
+    </ToolBoxCardHeader>
+    <ToolBoxCardBody :with-bg-color="withBgColor" class="break-all">
+      <span class="block w-full">
+        {{ url }}
+      </span>
+      <ToolBoxCardHeaderButton
           :aria-label="t('datasets.toolBox.exportDatasets.apiDatasets.iconAlt')"
-          @icon-click="copyUrl"
+      >
+        <ButtonCustom
+            class="mt-4 flex items-center gap-3 px-2 py-1.5 text-sm"
+            :size="Size.xs"
+            :aria-label="t('datasets.toolBox.exportDatasets.download.iconAlt')"
+            @click="copyUrl"
         >
-          <IconCopy v-if="!copiedUrl" />
+          <IconCopy v-if="!copiedUrl" class="size-4" />
           <IconCheck v-else />
-        </ToolBoxCardHeaderButton>
-      </ToolBoxCardHeader>
-      <ToolBoxCardBody :with-bg-color="withBgColor" class="break-all">{{
-        url
-      }}</ToolBoxCardBody>
-    </ToolBoxCard>
-
-    <DownloadCard v-if="url != null" :url="url"></DownloadCard>
-
-    <ToolBoxSectionLabel v-if="!!referencesUrls">{{
-      t('datasets.toolBox.exportDatasets.sectionReferencesData')
-    }}</ToolBoxSectionLabel>
-    <ToolBoxCard
-      v-for="referenceUrl in referencesUrls || []"
-      :key="getReferenceKey(referenceUrl)"
+          <span>{{ t('datasets.toolBox.exportDatasets.apiDatasets.button') }}</span>
+        </ButtonCustom>
+      </ToolBoxCardHeaderButton>
```

Comment:
Caution, you're putting here a button (`ButtonCustom`) inside a button (`ToolBoxCardHeaderButton`). I think you can remove the surrounding `ToolBoxCardHeaderButton`